src/components/routingActions.js

```diff
@@ -0,0 +1,14 @@
+export const getRedirectPathname = (action, actionParams) => {
+  switch (action) {
+    case 'cosign':
+      return `cosign?submission_uuid=${actionParams.get('submission_uuid')}`;
+    case 'afspraak-annuleren':
+      return 'afspraak-annuleren';
+    case 'afspraak-maken':
+      return 'afspraak-maken';
+    case 'stap':
```

Comment:
'stap' is not a meaningful action, instead please use `resume` as the action, and the action param can be `stepSlug`